egs/sad_rats/s5/local/make_mfcc.sh

```diff
@@ -0,0 +1,194 @@
+#!/usr/bin/env bash
```

Comment:
Could you mention how this script is different from `steps/make_mfcc.sh` and why it is needed? You can add this description in the comments at the top.

Reply:
Since extracting segment-level features is not necessary, I removed this script.